From: Carsten Schlote <c.schlote@konzeptpark.de>
Date: Tue, 1 Apr 2008 15:49:37 +0200
Subject: [PATCH] [gdb-6.8] Updated rules files and fixed code for m68k native GDB

Hmm, nobody seems to build GDB for Coldffire nativly. Changed call
to reg_regcache_arch() to get_regcache_arch().

It fixes the compile, but is of course untested and may break GDB on
your target.

Signed-off-by: Carsten Schlote <c.schlote@konzeptpark.de>

Index: gdb-6.8/gdb/m68klinux-nat.c
===================================================================
--- gdb-6.8.orig/gdb/m68klinux-nat.c	2008-04-01 15:34:54.000000000 +0200
+++ gdb-6.8/gdb/m68klinux-nat.c	2008-04-01 15:43:19.000000000 +0200
@@ -179,7 +179,7 @@
 static void
 store_register (const struct regcache *regcache, int regno)
 {
-  struct gdbarch *gdbarch = reg_regcache_arch (regcache);
+  struct gdbarch *gdbarch = get_regcache_arch (regcache);
   /* This isn't really an address.  But ptrace thinks of it as one.  */
   CORE_ADDR regaddr;
   char mess[128];		/* For messages */
